### PR TITLE
Modified services and configs to re-route traffic through edge-proxy

### PR DESCRIPTION
**Pull request by psppalash**
Changing Devicedb, ~Devicejs~, Relay-term, maestro config so that all services on raspberry Pi can talk through edge-proxy.
Tested functionality of relay-term, maestro by running edge-proxy on raspberry Pi.
~Needs further testing when edge-proxy is added to the meta-pelion-edge repo.~

### CODE REVIEW

**Comment by psppalash** [comment]
DO NOT MERGE until fp-edge service is added to meta-pelion-edge.

File: recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf [hunk 1]
```diff
@@ -182,8 +182,8 @@ cloud:
     # historyID: "*.wigwag.io"
     # historyHost and historyPort may be ommitted. In this case historyHost and
     # historyPort are set to the normal cloud host and port
-    historyURI: {{ARCH_GW_SERVICES_URL}}/relay-history/history
-    alertsURI: {{ARCH_GW_SERVICES_URL}}/relay-alerts/alerts
+    historyURI: http://gateways.local:8080/relay-history/history
```
Comment (ygoyal18) [inline recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf:185]: @psppalash Can we create a var_defs in maestro and define the fog-proxy local address - "http://gateways.local:8080" there and use mustache in the template files?

Comment (psppalash) [reply recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf:185]: Will do that

**Comment by psppalash** [comment]
Need to rebase against master. And get Rpi build to test the changes.

**Comment by psppalash** [comment]
Need to update maestro to latest SHA in this PR

**Comment by psppalash** [comment]
Tested on Rpi. Verified logs and terminal work through the portal. This PR is ready to be merged.

**Review by costanic — DISMISSED** [review]

**Comment by ygoyal18** [comment]
@costanic @x85446  This is ready to be merged. I have rebased to dev. This will make relay-term, maestro and devicedb communicate to cloud via edge-proxy.

File: recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf [hunk 2]
```diff
@@ -182,8 +182,8 @@ cloud:
     # historyID: "*.wigwag.io"
     # historyHost and historyPort may be ommitted. In this case historyHost and
     # historyPort are set to the normal cloud host and port
-    historyURI: {{ARCH_GW_SERVICES_URL}}/relay-history/history
-    alertsURI: {{ARCH_GW_SERVICES_URL}}/relay-alerts/alerts
+    historyURI: http://{{FOG_PROXY_ADDR}}/relay-history/history
```
Comment (cmonr) [inline recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf:185]: Double checking my understanding.

This, and other URLs, aren't https because this is all localhost, correct?

File: recipes-wigwag/maestro/maestro_0.0.1.inc [hunk 3]
```diff
@@ -54,7 +54,7 @@ file://maestro-watcher.path \
 "
 
 SRCREV_FORMAT="m"
-SRCREV_m="d6ab0da4238bb4d8b767db3f70af71db909fae1c"
+SRCREV_m="458a91f7278e8431ab446c3b751500e2b390c262"
```
Comment (cmonr) [inline recipes-wigwag/maestro/maestro_0.0.1.inc:57]: What do these SHAs point to?

Since https://github.com/PelionIoT/maestro/pull/132 is still unmerged, won't this above SHA change once it's merged?

File: recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf (hunk 2 above)
Comment (ygoyal18) [reply recipes-wigwag/maestro/maestro/rpi3/devicedb.template.conf:185]: Yes, thats correct.

File: recipes-wigwag/maestro/maestro_0.0.1.inc (hunk 3 above)
Comment (ygoyal18) [reply recipes-wigwag/maestro/maestro_0.0.1.inc:57]: Yes, we decided to lock the SHA to the latest release of the component. That SHA is pointing to the master branch and v2.7.0 release of maestro - https://github.com/PelionIoT/maestro/releases/tag/v2.7.0

**Review by costanic — APPROVED** [review]
